Evaluation.html

```diff
@@ -5,23 +5,22 @@
 </head>
 <body>
 	<header>
-		<ul class="pageHeader">
-			<li >
+		<ul class="header headerT">
```

Comment:
Theme , the menthor that came today proposed us to divide our css code into thematical css classes that work with color text size and overall feel of the button like a skin  , and localion wise that modifies the position and height stuff about the place and size